<testsuites id="" name="" tests="1" failures="1" skipped="0" errors="0" time="35.509873">
<testsuite name="e2e\employees\edit-employee-info.spec.ts" timestamp="2025-03-17T02:16:51.054Z" hostname="chromium" tests="1" failures="1" skipped="0" time="30.467" errors="0">
<testcase name="[US02] Gestión de empleados | Editar empleados correctamente › Validar editar un empleado exitosamente" classname="e2e\employees\edit-employee-info.spec.ts" time="30.467">
<failure message="edit-employee-info.spec.ts:10:9 Validar editar un empleado exitosamente" type="FAILURE">
<![CDATA[  [chromium] › e2e\employees\edit-employee-info.spec.ts:10:9 › [US02] Gestión de empleados | Editar empleados correctamente › Validar editar un empleado exitosamente › Entonces el empleado se encuentra en la lista de empleados 

    Test timeout of 30000ms exceeded.

    Error: expect(locator).toBeAttached()

    Locator: getByRole('cell', { name: '2579' }).first()
    Expected: attached
    Received: <element(s) not found>
    Call log:
      - expect.toBeAttached with timeout 10000ms
      - waiting for getByRole('cell', { name: '2579' }).first()


      83 |             const locator = employeePage.idColumnValues(newUniqueID);
      84 |             await employeePage.page.screenshot({ path: 'search_result.png' });
    > 85 |             await expect(locator).toBeAttached({ timeout: 10000 });
         |                                   ^
      86 |         });
      87 |     });
      88 | });
        at C:\Programacion\QA\Automation\Playwright-ts\Proyecto OrangeHRM\OrangeHRM-Demo\tests\e2e\employees\edit-employee-info.spec.ts:85:35
        at C:\Programacion\QA\Automation\Playwright-ts\Proyecto OrangeHRM\OrangeHRM-Demo\tests\e2e\employees\edit-employee-info.spec.ts:80:9
]]>
</failure>
</testcase>
</testsuite>
</testsuites>